docs/contributors/advanced/plus-one-maintenance.md

```diff
@@ -1,3 +1,3 @@
-(contributor-plus-one-maintenance)=
+(plus-one-maintenance)=
```

Comment:
page is still empty and this is not a MIR page
but ok, you only changed the header which is fine

And I see it pop up in the TOC above

Reply:
This is to fix a build error (otherwise RTD complains)